src/core/helpers/svgBackgroundImage.ts

```diff
@@ -1,5 +1,5 @@
 import svgToTinyDataUri from 'mini-svg-data-uri';
 
-export const svgImageUrl = (svg: string): string => {
```

Comment:
There's a case to be made for this to live somewhere more accessible, such as [`@guardian/src-foundations/utils`](https://github.com/guardian/source/tree/main/src/core/foundations/src/utils) or even [`@guardian/libs`](https://github.com/guardian/libs). It could probably use some documentation. What do you think?

Reply:
We’ve already move it upstream in 859e7bae057de48ec2d106f6a545ec18c391f6be. If there’s a desire to move it to [`@guardian/libs`][], I’m happy to move it there. Not sure how common of a use case this would be, as hopefully icons come from Source anyways?

[`@guardian/libs`]: https://github.com/guardian/libs

Reply:
Agreed, thanks @mxdvl I think `src-helpers` is an appropriate place for this for now.